python-cffi/netplan/netdef.py

```diff
@@ -315,6 +317,8 @@ def __str__(self):
             route = route + f' metric {self.metric}'
         if self.table > self._TABLE_UNSPEC_:
             route = route + f' table {self.table}'
+        if self.advmss > self._ADVMSS_UNSPEC_:
+            route = route + f' advmss {self.advmss}'
```

Comment:
I wanted to implement showing advmss in `netplan status`, but i not shure about practise needs of this information. Comparison of routes should be done without MCC, since there cannot be a route to the same network through the same gateway but with a different MCC. I removed unused code in NetplanRoute class